cmd/tester/Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM golang:1.9.2 as builder
```

Comment:
Copy the example from the vcenter connector please it has an updated Golang version.